src/utils/SVG.cpp

```diff
@@ -45,6 +46,12 @@ SVG::SVG(const char* filename, AABB aabb, Point canvas_size)
         fprintf(out, "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n");
     }
     fprintf(out, "<svg xmlns=\"http://www.w3.org/2000/svg\" version=\"1.1\" style=\"width:%llipx;height:%llipx\">\n", canvas_size.X, canvas_size.Y);
+    
+    if(background != Color::NONE)
+    {
+        fprintf(out, "<rect width=\"100%\" height=\"100%\" fill=\"%s\"/>\n", toString(background).c_str());
+    }
```

Comment:
Adding fill-opacity is better way to solve this, so you have color and opacity instead.